src/main/webapp/WEB-INF/pages/add_transport.jsp

```diff
@@ -0,0 +1,145 @@
+<script src="http://malsup.github.com/jquery.form.js"></script> 
+<script src="/js/utils.js"></script>
+<script src="/js/address.js"></script>
+<script src="/js/client.js"></script>
+<script src="/js/transport.js"></script>
+<script src="/js/token.js"></script>
+<script>
+	$(function() {
+		$("#loadDate").datepicker({ dateFormat: 'yy-mm-dd' });
+	});
+</script>
+<script>
+	$(function() {
+		$("#unloadDate").datepicker({ dateFormat: 'yy-mm-dd' });
+	});
+</script>
+</head>
+
+<body>
+
+	<!-- Static navbar -->
+	<nav class="navbar navbar-inverse">
+		<div class="container-fluid">
+			<div class="navbar-header">
+				<button type="button" class="navbar-toggle collapsed"
+					data-toggle="collapse" data-target="#navbar" aria-expanded="false"
+					aria-controls="navbar">
+					<span class="sr-only">Toggle navigation</span> <span
+						class="icon-bar"></span> <span class="icon-bar"></span> <span
+						class="icon-bar"></span>
+				</button>
+				<a class="navbar-brand" href="/">logistics_mgmt</a>
+			</div>
+			<div id="navbar" class="navbar-collapse collapse">
+				<ul class="nav navbar-nav">
+					<li class="inactive"><a href="/drivers">Kierowcy</a></li>
+					<li class="inactive"><a href="/vehicles">Pojazdy</a></li>
+					<li class="active"><a href="/transports">Transporty</a></li>
+				</ul>
+			</div>
+			<!--/.nav-collapse -->
+		</div>
+		<!--/.container-fluid -->
+	</nav>
+
+	<form name="add_transport_form" id="add_transport_form"
+		modelAttribute="addTransportForm" data-toggle="validator"
+		onsubmit="return addFreightTransport();">
+
+
+		<div class="form-group">
+			<label for="loadAddressId">Adres załadunku:</label> <select
+				name="loadAddressId" id="loadAddressId" path="loadAddressId"
+				class="form-control">
+				<c:forEach var="loadAddress" items="${load_address}">
+					<option value=${loadAddress.id}>${loadAddress.street}
+						${loadAddress.houseNumber}, ${loadAddress.town},
+						${loadAddress.country}</option>
+				</c:forEach>
+			</select>
+		</div>
+
+		<div class="form-group">
+			<label for="unloadAddressId">Adres rozładunku:</label> <select
+				name="unloadAddressId" id="unloadAddressId" path="unloadAddressId"
+				class="form-control">
+				<c:forEach var="unloadAddress" items="${unload_address}">
+					<option value=${unloadAddress.id}>${unloadAddress.street}
+						${unloadAddress.houseNumber}, ${unloadAddress.town},
+						${unloadAddress.country}</option>
+				</c:forEach>
+			</select>
+		</div>
+
+		<div class="form-group">
+			<label for="clientId">Klient:</label> <select name="clientId"
+				id="clientId" path="clientId" class="form-control">
+				<c:forEach var="client" items="${clients}">
+					<option value=${client.id}>${client.name}</option>
+				</c:forEach>
+			</select>
+		</div>
+
+		
+
+		<div class="form-group">
+			<label for="value">Wartość:</label>
+			<div class="input-group">
+				<span class="input-group-addon"></span><input type="number" step="any"
```

Comment:
Empty `input-group-addon` span item, IMHO it should contain `zł` as it is done here: https://github.com/logistics-mgmt/logistics-mgmt/blob/master/src/main/webapp/WEB-INF/pages/add_driver.jsp#L67